### DISCUSSION TITLE
Compass bearing

### DISCUSSION THREAD

**Discussion by Mikkooooo** [BimmerGestalt/AAIdrive#689] [Q&A]
Hi,
I‘ve aaidrive running with sucess on my F15 X5   ID4++.
Many features are running fine but not the Spotify playlist in my HUD but it is caused by ID4 and I can live with it.
I have also running Homeassistant (sometimes it crashes).
The Mapbox shows a strange behaviour  - the compass bearing and the orientation of the map don‘t work properly until now.
Did anybody notice this behaviour too  ?
Many thanks to Hufman for all.
Frank

**Comment by hufman** [comment]
You are probably seeing #624. It seems some cars don't announce their compass orientation in the way AAIdrive expects.
Try the new beta build in the Play Store to check what compass direction the car reports through the new Car Info screen.

**Comment by Mikkooooo** [comment]
Thanks again Walter,
the new  created car info screen only shows
 - motor temperature                    - outside temperature
  - oil temperature                        -  inside temperature
                                                      - 10,0 % HVAC
  - tank volume                               
  -  battery level                              -  Gear P
 

There are no other infos .

Best regards
Frank

**Comment by Mikkooooo** [comment]
Attached a photo :


![IMG_20230413_191138](https://user-images.githubusercontent.com/64146978/231845776-80a3e3a4-1c2e-4faf-b1dd-56a28ed2eddb.jpg)

**Reply by hufman** [reply]
Ah I only pushed the new version to the store this morning and it normally takes a few days to roll out.
Please join the discussion in #624

**Reply by Mikkooooo** [reply]
Thanks Walter,
I hope I can try it out this weekend.
I have a F15  with a compass in the middle of the car dashboard.
May be that‘s the reason ?
BTW : did you get my PM where I‘ve asked you if I can donate over Paypal ???
Such a good work shall be rewarded !!!

**Reply by hufman** [reply]
I haven't yet figured out any pattern for what cars work or not.
I haven't seen any PM, but there's a BuyMeACoffee link on the main Readme and within the app.
